sig-scalability/charter.md

```diff
@@ -0,0 +1,110 @@
+# SIG Scalability Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README]
+and uses the Roles and Organization Management outlined in [sig-governance].
+
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+
+## Scope
+
+SIG Scalability primary responsibilities are to define scalability goals for
+Kubernetes, define, test and measure performance and scalability related
+Service Level Indicators (SLIs) and ensure that every Kubernetes release meets
+Service Level Objectives (SLOs) build on top of the above SLIs.
+
+We also coordinate and contribute to general system-wide scalability and
+performance improvements (that don't fall into the charter of another individual
+SIG) as well as provide consultations about any scalability and performance
+related aspects of Kubernetes.
+
+### In Scope
+
+#### Code, Binaries and Services:
+
+- Scalability and performance testing frameworks. Examples include:
+  - [Cluster loader](https://github.com/kubernetes/perf-tests/tree/master/clusterloader2)
+  - [Kubemark](https://github.com/kubernetes/kubernetes/tree/master/cmd/kubemark)
+- Scalability and performance tests:
+  - [Tests](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/scalability/)
+
+#### Cross-cutting and Externally Facing Processes
+
+- Defining what does it mean that “Kubernetes scales”. This includes defining
```

Comment:
I'd nix the 1st part and just outline the SLIs and SLOs.

Reply:
I think that the fact that this is the definition of "Kubernetes scalability" is relatively important.
I rephrased it a bit - ptal.